packages/assets-controllers/src/TokensController.test.ts

```diff
@@ -109,6 +110,8 @@ describe('TokensController', () => {
     tokenListStateChangeListener = listener;
   });
 
+  const fakeProvider = new FakeProvider();
```

Comment:
Heads up — it's best to avoid setting variables outside of `beforeEach`/`afterEach`/`it`, because it means that changes to an object could persist across multiple tests, which could lead to tests that intermittently fail or that are difficult to debug. In this case it's okay since the fake provider isn't actually used for anything, it's just here to satisfy a type, but something to keep in mind for future tests. This is something we go into more detail about in the [contributor docs](https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#reset-shared-variables), but we could certainly make this more clear if it's not clear enough.